CODEOWNERS

```diff
@@ -9,6 +9,7 @@
 /chain/jsonrpc-primitives/ @khorolets @miraclx @chefsale
 /chain/indexer/ @khorolets
 /chain/rosetta-rpc/ @frol @mina86
+/utils @pmnoxx
```

Comment:
please add me there too (as we should have at least 2 people for each directory here)